1/Bob.py

```diff
@@ -0,0 +1,12 @@
+phrase = input("say smth to Bob\n")
+
+if phrase.upper() == phrase:
+    print("Whoa, chill out!")
+elif phrase[len(phrase)-1] == '?':
```

Comment:
It's better to use [-1] or str method endswith() to get last element of the sequence